### ISSUE TITLE
QUEST Fate, Up Against Your Will Unplayable

### ISSUE THREAD

**Issue by Torrek12** [dalaranwow/dalaran-wow#4837]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When you enter the throne room and get the quest, Jaina Proudmoore is nowhere to be seen, and you can therefore not continue the quest. And no matter whether I leave Stormwind, return, log-off or retake the quest, Jaina is still not there.

**Expected behaviour**: Tell us what should happen instead
Jaina Proudmoore should stand to the left-hand side of King Varian Wrynn

**Steps to reproduce the problem**:

1. Finish the quest [Reborn From The Ashes]
2. Take the quest [Fate, Up Against Your Will]

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

http://wowwiki.wikia.com/wiki/Quest%3AFate%2C_Up_Against_Your_Will
https://www.youtube.com/watch?v=yXnAZZC6OTU

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![skaermbillede 2017-07-30 15 25 42](https://user-images.githubusercontent.com/30571100/28753910-6661457c-753b-11e7-9eb8-75bfbfa93950.png)

**Comment by Edzero** [comment]
This is already covered in #3870.

**Comment by TeridanDW** [comment]
> This is already covered in #3870.
